docs/use-cases/navigate.md

```diff
@@ -3,6 +3,53 @@ title: "Teleoperate and navigate rovers"
 linkTitle: "Teleoperate and navigate with rovers"
 weight: 60
 type: "docs"
-layout: "empty"
-canonical: "tutorials/services/navigate-with-rover-base/"
+tags: ["navigation", "motion", "services", "base", "base remote control"]
+no_list: true
+description: "Use Viam to teleoperate and navigate rover bases."
+image: "/services/icons/navigation.svg"
+imageAlt: "Navigation"
+images: ["/services/icons/navigation.svg"]
 ---
+
+If you have a rover base, you can use Viam to teleoperate it and to navigate autonomously.
+Once you have configured your machine, you can remotely control your machine on the app's **Control** tab, and set up autonomous navigation with the [navigation service](/mobility/navigation/).
+
+<table>
+  <tr>
+    <th>{{<imgproc src="/use-cases/base-control.png" class="fill alignright" resize="200x" style="max-width: 200px" declaredimensions=true alt="Base control card">}}
+      <b>1. Teleoperate</b>
+      <p>You can remotely control your rover from anywhere through the <a href="https://app.viam.com">Viam app.</a> Create an account and add a machine, <a href="/get-started/installation/">install viam-server</a>, and <a href="/components/base/">configure your rover base</a>. Then, go to the <b>Control</b> tab and access a remote control card for your base, with an interface for controlling speed, direction, and power. You can also view live feeds from any cameras you configure.</p>
+    </th>
+  </tr>
+  <tr>
+    <th>{{<imgproc src="/services/icons/base-rc.svg" class="fill alignleft" resize="200x" style="max-width: 200px" declaredimensions=true alt="Base remote control service icon.">}}
+      <b>2. Program to move</b>
+      <p>Remotely control your rover base programmatically with a <a href="/sdks/">Viam SDK.</a> Make calls to the <a href="/components/base/#api">base API</a> or <a href="/mobility/motion/">motion service</a>.
```

Comment:
@JessamyT @npentrel What do we think about motion service mention like this? It is hard to use but I agree w/ naomi it should still be on page